src/classes/ERR_Handler.cls

```diff
@@ -188,6 +188,7 @@ public with sharing class ERR_Handler {
 	    	}
 	                
 	    	if(UTIL_CustomSettingsFacade.getErrorSettings().Store_Errors_On__c) {
+	    	   UTIL_Debug.debug('**** Storing errorRecords: ' + errorRecords);
```

Comment:
We disabled debug statements in the managed package context, unless you set
a setting, so we're not affecting performance for that anymore. Maybe test
performance a bit, but I think that's negligible.

On Tue, Jan 27, 2015 at 3:01 PM, Kevin Bromer <notifications@github.com>
wrote:

> In src/classes/ERR_Handler.cls
> <https://github.com/SalesforceFoundation/Cumulus/pull/1317#discussion_r23651500>
> :
>
> > @@ -188,6 +188,7 @@ public with sharing class ERR_Handler {
> >  	    	}
> >  	
> >  	    	if(UTIL_CustomSettingsFacade.getErrorSettings().Store_Errors_On__c) {
> > +	    	   UTIL_Debug.debug('**** Storing errorRecords: ' + errorRecords);
>
> We need to look at the overhead at some point, but youre right, this
> probably won't be the straw that hurts our camel
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SalesforceFoundation/Cumulus/pull/1317/files#r23651500>
> .
>